docs/001_develop/01_development-environment/003_genesis-start/index.mdx

```diff
@@ -11,57 +11,59 @@ tags:
 sidebar_position: 3
 ---
 
-Genesis Start is a small program which allows you to preview an application, including during the development cycle, by running it on your local desktop. 
+Genesis Start enables you to run an application on your local desktop. This is useful when you are developing locally.
```

Comment:
Thanks for this. Where you see me use Preview its because marketing keep saying they want to use that term, happy with run as an edit but just FYI.

Reply:
OK. Good to know. I'll use preview as well...